src/plugins/dismissable-content/dismissable-content-en.hbs

```diff
@@ -10,8 +10,15 @@
+<p id="dismiss2" class="wb-dismissable">Vivamus vulputate lectus sit amet mauris venenatis, sed tempus lorem gravida. Praesent eu neque et ipsum luctus congue non in nulla. Morbi blandit libero sed urna congue dignissim. Praesent ut leo eu quam fringilla pellentesque. Morbi congue quam lacus, ut auctor est viverra pellentesque. Nullam augue risus, pharetra a justo id, vulputate efficitur arcu. Ut ut ligula ante. Aenean luctus tellus et eros lacinia aliquet. Fusce dictum, orci a bibendum ullamcorper, urna justo consectetur justo, sit amet maximus magna nisi mattis leo. Praesent luctus lacus nulla, id imperdiet arcu facilisis a.</p>
 <p>Aenean tempor porta leo a porttitor. Nulla sed libero velit. Aliquam gravida euismod ante vitae porttitor. Mauris id nisl sit amet felis semper iaculis a ac augue. Pellentesque nec aliquet ligula, et maximus ligula. Nam gravida lobortis sapien at interdum. Phasellus dictum erat id felis hendrerit maximus. Sed in lectus faucibus, finibus sapien nec, porttitor lectus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Maecenas ac felis et nisl rhoncus sagittis vitae ut velit. Suspendisse et leo cursus, interdum quam eu, efficitur nunc. Aliquam sit amet ipsum vitae augue consequat laoreet in nec turpis. Duis nec maximus neque. Etiam leo quam, tincidunt non tempor et, maximus at lacus.</p>
-<p class="wb-dismissable">Fusce malesuada massa eget ipsum consequat, ut mattis massa mattis. Ut auctor molestie molestie. Praesent in malesuada est. Nullam ullamcorper justo at nisi porta, at volutpat dui lobortis. Aliquam erat volutpat. Donec eget erat non mi sollicitudin blandit vitae vel nibh. Aliquam tempus nibh eu massa rhoncus vulputate ut non orci. Ut faucibus mattis eleifend.</p>
+<p id="dismiss3" class="wb-dismissable">Fusce malesuada massa eget ipsum consequat, ut mattis massa mattis. Ut auctor molestie molestie. Praesent in malesuada est. Nullam ullamcorper justo at nisi porta, at volutpat dui lobortis. Aliquam erat volutpat. Donec eget erat non mi sollicitudin blandit vitae vel nibh. Aliquam tempus nibh eu massa rhoncus vulputate ut non orci. Ut faucibus mattis eleifend.</p>
+<section>
+	<h2>Bring the dismissed content back (Demo only)</h2>
+	<p><strong>Note:</strong> To see the effect of these buttons, the page must be refreshed.</p>
+	<input type="button" value="Reset box 1" onclick="localStorage.removeItem( 'dismiss1' );" class="btn btn-primary">
```

Comment:
I'm just doing what the country content page is doing. And it's just a quick fix because I keep forgetting to clear local storage and think this plugin is broken all the time. 

Shawn Thompson 
Plansmash@me.com

> On Apr 10, 2016, at 12:09 PM, Mario Bonito <notifications@github.com> wrote:
> 
> In src/plugins/dismissable-content/dismissable-content-en.hbs:
> 
> >  <p>Aenean tempor porta leo a porttitor. Nulla sed libero velit. Aliquam gravida euismod ante vitae porttitor. Mauris id nisl sit amet felis semper iaculis a ac augue. Pellentesque nec aliquet ligula, et maximus ligula. Nam gravida lobortis sapien at interdum. Phasellus dictum erat id felis hendrerit maximus. Sed in lectus faucibus, finibus sapien nec, porttitor lectus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Maecenas ac felis et nisl rhoncus sagittis vitae ut velit. Suspendisse et leo cursus, interdum quam eu, efficitur nunc. Aliquam sit amet ipsum vitae augue consequat laoreet in nec turpis. Duis nec maximus neque. Etiam leo quam, tincidunt non tempor et, maximus at lacus.</p>
> > -<p class="wb-dismissable">Fusce malesuada massa eget ipsum consequat, ut mattis massa mattis. Ut auctor molestie molestie. Praesent in malesuada est. Nullam ullamcorper justo at nisi porta, at volutpat dui lobortis. Aliquam erat volutpat. Donec eget erat non mi sollicitudin blandit vitae vel nibh. Aliquam tempus nibh eu massa rhoncus vulputate ut non orci. Ut faucibus mattis eleifend.</p>
> > +<p id="dismiss3" class="wb-dismissable">Fusce malesuada massa eget ipsum consequat, ut mattis massa mattis. Ut auctor molestie molestie. Praesent in malesuada est. Nullam ullamcorper justo at nisi porta, at volutpat dui lobortis. Aliquam erat volutpat. Donec eget erat non mi sollicitudin blandit vitae vel nibh. Aliquam tempus nibh eu massa rhoncus vulputate ut non orci. Ut faucibus mattis eleifend.</p>
> > +<section>
> > +	<h2>Bring the dismissed content back (Demo only)</h2>
> > +	<p><strong>Note:</strong> To see the effect of these buttons, the page must be refreshed.</p>
> > +	<input type="button" value="Reset box 1" onclick="localStorage.removeItem( 'dismiss1' );" class="btn btn-primary">
> @shawnthompson - do you think it is the best idea to leverage the onclick attribute to clear localstorage instead of passing a data-attribute or the likes? I am leaning towards a more native JS method so that we ensure a proper removal as this feature grows.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>